### ISSUE TITLE
Issue about createReply

### ISSUE THREAD

**Issue by wytang1234** [arjunkomath/node-freshdesk-api#144]
##npm version
 freshdesk-api V2

## Current behavior
I try to create fresh desk reply with the api with this config:
```
freshdesk.createReply(123, {
    body: `Testing at ${new Date()}`,
    attachments: [
      fs.createReadStream('xxx.xlsx'),
      fs.createReadStream('sample.json')
    ],
    cc_emails: ["yu-v-@hotmail.com"], 
    // bcc_emails: ["xxxx@xxx"]
  }, function (err, data) {
      console.log(err || data)
  })
```

However its returning error message:
```
Error: form-data: Arrays are not supported.
    at FormData.append (/Users/yu/Desktop/test/node_modules/request/node_modules/form-data/lib/form_data.js:66:17)
    at makeRequest (/Users/yu/Desktop/test/node_modules/freshdesk-api/lib/utils.js:200:9)
    at Freshdesk.createReply (/Users/yu/Desktop/test/node_modules/freshdesk-api/lib/client.js:155:3)
    at main (/Users/yu/Desktop/test/index.js:75:13)
    at Object.<anonymous> (/Users/yu/Desktop/test/index.js:87:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```
just check the problem on cc_emmails and bcc_emails
For the [API Doc](https://developers.freshdesk.com/api/#reply_ticket), bcc_emails and cc_emails allow to use
how can I change my api setting?

BTW, I find it is a working setting when I use unirest
 ```
// unirest.post(URL)
  //   .headers(headers)
  //   .field('body', `Testing at ${new Date()}`)
  //   .field('cc_emails[]', "yyy@yyy.com")
  //   .field('bcc_emails[]', "xxx@xxx.com")
  //   .attach('attachments[]', fs.createReadStream('xxx.xlsx'))
  //   .attach('attachments[]', fs.createReadStream('sample.json'))
  //   .end(function(response){ console.log(response.req.res.body) })
```

Thanks!

**Comment by arjunkomath** [comment]
Hey @wytang1234 

I believe there is a mistake in the documentation, the attachments array should be an object. Could you try changing it to:
```js
freshdesk.createReply(123, {
    body: `Testing at ${new Date()}`,
    attachments: [
      {'test1': fs.createReadStream('xxx.xlsx')},
      {'test2': fs.createReadStream('sample.json')}
    ],
  }, function (err, data) {
      console.log(err || data)
  })
```

**Comment by wytang1234** [comment]
@arjunkomath i think no...
its working for me and can send attachments if:
`
freshdesk.createReply(123, {
    body: `Testing at ${new Date()}`,
    attachments: [
      fs.createReadStream('xxx.xlsx'),
      fs.createReadStream('sample.json')
    ]
  }, function (err, data) {
      console.log(err || data)
  })
`

so i think it is problem on cc_emails and bcc_emails since those are array type.

**Comment by mloenow** [comment]
@wytang1234, @arjunkomath - this is somewhat similar to what we experienced with `custom_fields` and form data, which I added support for in https://github.com/arjunkomath/node-freshdesk-api/pull/147. It switches automatically from `application/json` to `multipart/form-data` when attachments are included.

I think the else clause should check the value with `Array.isArray()` and if so append it one by one:

```
...
 		} else {
 		 	if (Array.isArray(data[key])) {
				for (let i = 0; i < data[key].length; i++) {
					form.append(key + '[]', data[key][i])
				}
 			} else {
 				form.append(key, data[key])
			}
		}
...
```

This may actually support attachments also, so perhaps all the code could be simplified further.
